src/cmd/set.rs

```diff
@@ -124,14 +125,21 @@ impl Set {
     ///
     /// The response is written to `dst`. This is called by the server in order
     /// to execute a received command.
-    #[instrument(skip(self, db, dst))]
+    #[instrument(
+        level = "trace",
+        name = "Set::apply",
+        skip(self, db, dst),
+        fields(
+            key = self.key.as_str(),
+            ?expire = self.expire.as_ref().map(Duration::as_secs_f64),
```

Comment:
should be
```suggestion
            expire = ?self.expire.as_ref().map(Duration::as_secs_f64),
```
the question mark goes before the value that should be formatted with `Debug`

Reply:
also, this doesn't include a unit, so it should emit things like
```
expire = Some(1.45)
```
which isn't very helpful.

i would probably either use the `fmt::Debug` impl for `Duration` (which automatically determines the best unit) directly, like this:
```suggestion
            expire = ?self.expire.as_ref(),
```
or, consider renaming the field name to include a unit:
```suggestion
            expire_secs = ?self.expire.as_ref().map(Duration::as_secs_f64),
```

personally, i prefer the first option.